src/components/login/Login.tsx

```diff
@@ -0,0 +1,109 @@
+import { Button, Dialog, DialogActions, DialogContent, DialogContentText, DialogTitle, TextField } from '@material-ui/core';
+import * as React from 'react';
+import * as rest from "../../rest/interface";
+import '../../styles.css';
+
+interface State {
+  open : boolean;
+  submitted : boolean
+  username : string
+  password : string
+}
+
+class Props {
+  public client: rest.Evergreen;
+}
+
+export class Login extends React.Component<Props, State> {
+
+  constructor(props: Props) {
+    super(props);
+    this.handleClickOpen = this.handleClickOpen.bind(this);
+    this.handleClickClose = this.handleClickClose.bind(this);
+    this.handleClickSubmit = this.handleClickSubmit.bind(this);
+    this.state = {
+      open : false,
+      submitted : false,
+      username : "",
+      password : "",
+    };
+  }
+
+  public handleClickOpen() : void {
```

Comment:
It doesn't make sense for any other component to call these functions, so they should be private

Reply:
Fixed